Reconstruction/RecCalorimeter/tests/options/runFullCaloSystem_SimAndDigitisation.py

```diff
@@ -21,7 +21,8 @@
                                            'file:Detector/DetFCChhHCalTile/compact/FCChh_HCalBarrel_TileCal.xml',
                                            'file:Detector/DetFCChhHCalTile/compact/FCChh_HCalExtendedBarrel_TileCal.xml',
                                            'file:Detector/DetFCChhCalDiscs/compact/Endcaps_coneCryo.xml',
-                                           'file:Detector/DetFCChhCalDiscs/compact/Forward_coneCryo.xml'
+                                           'file:Detector/DetFCChhCalDiscs/compact/Forward_coneCryo.xml',
+                                           'file:Detector/DetFCChhTailCatcher/compact/FCChh_TailCatcher.xml'
```

Comment:
TailCatcher does not belong to testing of the calorimetry.
One can create a separate test for it.